app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -3,6 +3,7 @@ CONFIG_IPC_MAJOR_4=y
 
```

Comment:
Can you please add some justification in the commit message of why this patch is needed?

Reply:
@fkwasowi e.g. best to say we are saving X MCPS or Y memory vs standard config SRC.

Reply:
Describe added.